lib/metasploit/framework/login_scanner/snmp.rb

```diff
@@ -17,6 +17,35 @@ class SNMP
         PRIVATE_TYPES        = [ :password ]
         REALM_KEY            = nil
 
+        # @!attribute retries
```

Comment:
You should separate your attributes from your validations, so that all attributes are first, then all validations as the patten shown [here](https://github.com/rapid7/metasploit-cache/blob/866e3b7f2d48000df0a41df86fb6be3efbe7a4d9/lib/metasploit/cache/direct/class/load.rb#L3-L53)